app/src/main/java/com/termux/app/ExtraKeysView.java

```diff
@@ -71,9 +75,15 @@ static void sendKey(View view, String keyName) {
             case "↓":
                 keyCode = KeyEvent.KEYCODE_DPAD_DOWN;
                 break;
+            case "↲":
+                keyCode = KeyEvent.KEYCODE_ENTER;
+                break;
             case "―":
                 chars = "-";
                 break;
+            case "-":
+                chars = "-";
+                break;
```

Comment:
Could be refactored:

```java
case "−":
case "-":
    // if it's "−" or "-", it's the same output
    chars = "-";
    break;
```

Including the comment, not so much people know the syntax.